Docs/source/usage/parameters.rst

```diff
@@ -1452,6 +1452,17 @@ Grid initialization
     the field solver. In particular, do not use any other boundary condition
     than periodic.
 
+* ``warpx.maxlevel_extEMfield_init`` (default is maximum number of levels in the simulation)
```

Comment:
Can you pls add this to PICMI while we add it, too? :)

Reply:
Sure. I ll discuss with @dpgrote on how to do this!

Reply:
done! 
Thanks to @dpgrote for the offiline discussion